.travis.yml

```diff
@@ -110,8 +110,11 @@ script:
 
   # QEMU-virt (PLATFORM=vexpress-qemu_virt)
   - $make
-  - $make CFG_TEE_CORE_LOG_LEVEL=4 DEBUG=1
-  - $make CFG_TEE_CORE_LOG_LEVEL=0 CFG_TEE_TA_LOG_LEVEL=0 DEBUG=0
+  - $make CFG_TEE_CORE_LOG_LEVEL={4,3,2,1,0} DEBUG=1
```

Comment:
```
The command "$make" exited with 0.
19.73s$ $make CFG_TEE_CORE_LOG_LEVEL={4,3,2,1,0} DEBUG=1
```

well... i this not what I was expecting...
i must say i'm a bit lazy and tried this pushing it straight in the PR.
i will split that in several lines to insure optee is built with each of the valid log levels.